packages/service/pyproject.toml

```diff
@@ -43,11 +43,11 @@ nidmm = { version = ">=1.4.4", extras = ["grpc"], optional = true }
 nifgen = { version = ">=1.4.4", extras = ["grpc"], optional = true }
 niscope = { version = ">=1.4.4", extras = ["grpc"], optional = true }
 niswitch = { version = ">=1.4.4", extras = ["grpc"], optional = true }
-ni-measurementlink-measurement-v1-proto = { version = ">=0.1.0dev0", allow-prereleases = true }
-ni-measurementlink-measurement-v2-proto = { version = ">=0.1.0dev0", allow-prereleases = true }
-ni-measurementlink-pinmap-v1-client = { version = ">=0.1.0dev0", allow-prereleases = true }
-ni-measurementlink-sessionmanagement-v1-client = { version = ">=0.1.0dev2", allow-prereleases = true }
-ni-protobuf-types = { version = ">=0.1.0dev2", allow-prereleases = true }
+ni-measurementlink-measurement-v1-proto = { version = "^1.0.0" }
```

Comment:
This package directly uses discovery client and grpc extensions, so I think they should be listed as direct dependencies. Please add them in main too.